products/revenue_analytics/backend/models.py

```diff
@@ -194,9 +194,15 @@ def for_schema_source(source: ExternalDataSource) -> Optional["RevenueAnalyticsR
             ),
         )
 
+        if source.prefix is None:
+            name = f"stripe.revenue_view"
+        else:
+            prefix = source.prefix.strip("_")
+            name = f"stripe.{prefix}.revenue_view"
```

Comment:
It's always stripe for now, so I don't mind hardcoding this, will eventually need to make this a little bit more dynamic